src/app/src/test/kotlin/org/imperial/mrc/hint/database/VersionRepositoryTests.kt

```diff
@@ -71,6 +73,25 @@ class VersionRepositoryTests
         assertThat(version[PROJECT_VERSION.VERSION_NUMBER]).isEqualTo(1)
     }
 
+    @Test
+    fun `can save version note`()
```

Comment:
I think there are missing tests here for notes saved during copyVersion, saveVersion and cloneVersion